CHANGELOG/unreleased/kong/11308.yaml

```diff
@@ -0,0 +1,7 @@
+message: Add a Request ID (which matches the trace_id or correlation_id when available) to error templates and logs.
```

Comment:
Should have another change log entry for lua-kong-module version bump.

The file could be `11308-2.YAML`